rcnn/CMakeLists.txt

```diff
@@ -2,13 +2,13 @@ cmake_minimum_required(VERSION 3.1)
 
 project(rcnn)
 
-add_definitions(-std=c++14)
```

Comment:
C++14, seems to be a must in my environment, otherwise brings more warnings

Reply:
For me, c++11 is also a must in my env. BTW, this is not a fixed param and can be specified by users. I will change it back to c++14 for your test.